src/public/app/widgets/highlighted_text.js

```diff
@@ -0,0 +1,267 @@
+/**
+ * Widget: Show highlighted text in the right pane
+ *
+ * By design there's no support for nonsensical or malformed constructs:
+ * - For example, if there is a formula in the middle of the highlighted text, the two ends of the formula will be regarded as two entries
+ */
+
+import attributeService from "../services/attributes.js";
+import RightPanelWidget from "./right_panel_widget.js";
+import options from "../services/options.js";
+import OnClickButtonWidget from "./buttons/onclick_button.js";
+
+const TPL = `<div class="highlighted-text-widget">
+    <style>
+        .highlighted-text-widget {
+            padding: 10px;
+            contain: none; 
+            overflow: auto;
+            position: relative;
+        }
+        
+        .highlighted-text > ol {
+            padding-left: 20px;
+        }
+        
+        .highlighted-text li {
+            cursor: pointer;
+            margin-bottom: 3px;
+            text-align: justify;
+            text-justify: distribute;
+        }
+        
+        .highlighted-text li:hover {
+            font-weight: bold;
+        }
+        
+        .close-highlighted-text {
+            position: absolute;
+            top: 2px;
+            right: 2px;
+        }
+    </style>
+
+    <span class="highlighted-text"></span>
+</div>`;
+
+export default class HighlightTextWidget extends RightPanelWidget {
+    constructor() {
+        super();
+
+        this.closeHltButton = new CloseHltButton();
+        this.child(this.closeHltButton);
+    }
+
+    get widgetTitle() {
+        return "Highlighted Text";
+    }
+
+    isEnabled() {
+        return super.isEnabled()
```

Comment:
This is a promising future, but it may also cause performance issues, especially on a poor machine like mine. Additionally, some users may not be accustomed to using the highlight feature. Personally I prefer to use bold to emphasize a snippet. So it would be beneficial to make the feature configurable for certain users.

Reply:
I think bold is often used to emphasize some keywords instead of sentences. I feel that it is more appropriate to only display highlights. I am not sure whether it is necessary to configure bold display

Reply:
oh I didnt see that. it's already pretty good. I dont think it's necessary to include bolded texts. 
Actually I prefer bolding just because it has a shortcut and is supported by markdown. In my patten,  bold, italic, quote texts are colored via custom styles, and these colors resonate with the trilium logo :)

Reply:
Modified to support bold italic underline
![图片](https://github.com/zadam/trilium/assets/37627919/275f09d0-e00d-47e8-8592-82baef1c67ea)